### PR TITLE
Jgithubusers

### PR DESCRIPTION
**Pull request by drmmr763**
This PR is to add a class for getting the github user profile. It's my first "bigger" PR to the platform, so I'm completely open to changing things as needed / recommended.

### CODE REVIEW

**Comment by dongilbert** [comment]
You should squash these commits so the old items don't appear in the history.

**Comment by elinw** [comment]
http://developer.joomla.org/pulls/pulls/1780.html

You need to tell your editor to trim trailing white space :P

**Comment by drmmr763** [comment]
I saw that once the pull tester ran. Updating soon!

On Thursday, January 3, 2013, elinw wrote:

> http://developer.joomla.org/pulls/pulls/1780.html
> 
> You need to tell your editor to trim trailing white space :P
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/joomla/joomla-platform/pull/1780#issuecomment-11867411.

## 

Regards,
Chad Windnagle
Fight SOPA https://www.google.com/landing/takeaction/

File: build/phpcs/Joomla [hunk 1]
```diff
@@ -1 +0,0 @@
-Subproject commit 893dc8b9e4bab85f933076ec1fb618a2c492427f
```
Comment (ianmacl) [inline build/phpcs/Joomla:1]: You need to remove this from your changeset.

File: joomla-platform-examples [hunk 2]
```diff
@@ -0,0 +1 @@
+Subproject commit 4beaa7a5c17b394e27dd961801ec4fc603a34c9c
```
Comment (ianmacl) [inline joomla-platform-examples:1]: Same with this.

**Comment by ianmacl** [comment]
You need unit tests for this code.

**Comment by drmmr763** [comment]
I'm going to close this for now and reopen once I get all this mess cleaned up.

**Pull request closed without merging**